lib/mllp/client.ex

```diff
@@ -0,0 +1,444 @@
+defmodule MLLP.ClientContract do
```

Comment:
For some reason  git is  being stubborn about marking this as a new  file vs moved and  changed. I can try to fix  it, let me know.